src/components/EmojiPicker.vue

```diff
@@ -68,6 +68,8 @@ export default defineComponent({
 
 .emoji-picker {
   border-radius: 8px;
+  position: absolute;
+  bottom: 0;
```

Comment:
Does `<v-menu location="top"` works without these styles?

Reply:
Not in Google Chrome on iOS